exercises/concept/roll-the-die/RollTheDieTests.cs

```diff
@@ -1,25 +1,38 @@
 using Xunit;
 using Exercism.Tests;
+using System.Collections.Generic;
 
 public class RollTheDieTests
 {
     [Fact]
     [Task(1)]
     public void RollDie()
     {
+        var rollCount = 1000;
+        var rolls = new HashSet<int>(rollCount);
         var player = new Player();
-        for (var i = 0; i < 100; i++)
+        for (var i = 0; i < rollCount; i++)
         {
+            var roll = player.RollDie();
+            rolls.Add(roll);
             Assert.InRange(player.RollDie(), 1, 18);
         }
+        Assert.Equal(18, rolls.Count);
     }
 
     [Fact]
     [Task(2)]
     public void GenerateSpellStrength()
     {
+        var rollCount = 100;
+        var rolls = new HashSet<double>(rollCount);
         var player = new Player();
-        var strength = player.GenerateSpellStrength();
-        Assert.InRange(strength, 0.0, 100.0);
+        for (var i = 0; i < rollCount; i++)
+        {
+            var strength = player.GenerateSpellStrength();
+            rolls.Add(strength);
+            Assert.InRange(strength, 0.0, 100.0);
+        }
+        Assert.Equal(rollCount, rolls.Count);
```

Comment:
Does this mean that you expect all 100 rolls to be unique, correct? If so, I wonder if we should maybe be slightly more tolerant of accidental duplicates. Maybe something like `Assert.True(rolls.Count >= rollCount - 5)` or something like that